gui/README.md

```diff
@@ -0,0 +1,37 @@
+# Cassy
+
+## Project setup
+```
+npm install
+```
+
+### To run the GUI with Cassy
+See the [Getting Started](../docs/getting-started.md) to set up your Cassy configuration and run the server.
+
+Once you have started Cassy, go to the root folder and run the following command to start the HTTP/1.1 server.
+```
+./entrypoint -dev
```

Comment:
I think this is slightly better. (having `dev` and `prod` maybe ?)
```suggestion
./entrypoint -mode=dev
```

Reply:
In its current state, the default mode is `prod`, so you can run the file by just doing `./entrypoint` . Would you still prefer the configuration above?